src/software/simulation/simulator.cpp

```diff
@@ -11,8 +12,29 @@ extern "C"
 }
 
 Simulator::Simulator(const World& world)
-    : physics_world(world), friendly_goalie_id(world.friendlyTeam().getGoalieID())
+    : physics_world(world.field()), friendly_goalie_id(world.friendlyTeam().getGoalieID())
 {
+    physics_world.setBallState(world.ball().currentState().ballState());
+    // Note: The simulator currently makes the invariant that friendly robots
+    // are yellow robots, and enemies are blue. This will be fixed in
+    // https://github.com/UBC-Thunderbots/Software/issues/1325
```

Comment:
Annnddd just deleted another comment above. Thanks for pre-emptively commenting about this :smile: